Week 1/Homework/calculator.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+if(CommandLine.arguments.count == 3){
+	var amountOfMoney:Float =  Float(CommandLine.arguments[1])!
+	var amountOfMonths:Float =  Float(CommandLine.arguments[2])!
+	var interestRate:Float =  Float(arc4random_uniform(15) + 5)
+	print("Interest Rate: " + String(Int(interestRate)) + "%")
+	for i in 0...Int(amountOfMonths){
+		let amountOfDollars = Int(amountOfMoney)
+		let amountOfCents = Int((amountOfMoney.truncatingRemainder(dividingBy:1.0)) * 100.0)
+		let amountOfDollarsString = String(amountOfDollars)
+		var amountOfCentsString = String(amountOfCents)
+		if(amountOfCents < 10){
+			amountOfCentsString = "0" + String(amountOfCents)
+		}
+		var timePassed = String(i) + " months: "
+		if(i == 1){
+			timePassed = "1 month: "
+		}
+		let fullMoneyString = "$" + amountOfDollarsString + "." + amountOfCentsString
+		print(timePassed + fullMoneyString)
+		amountOfMoney = amountOfMoney + (amountOfMoney * (interestRate/100))
+	}
+}
+else{
+	print("Enter 2 arguments in the order (amount of money, amount of months)")
```

Comment:
I like the error checks. I think you are missing an argument from my sample input.